FILE paseo.json
RANGE {"path":"paseo.json","old_lines":[4,7],"new_lines":[4,13],"diff_block_lines":[5,15],"hunk_header":"@@ -4,4 +4,10 @@","substantive_changed_lines":3}
@@ -4,4 +4,10 @@
       "npm ci",
       "cp \"$PASEO_SOURCE_CHECKOUT_PATH/packages/server/.env\" \"$PASEO_WORKTREE_PATH/packages/server/.env\""
+    ],
+    "terminals": [
+      {
+        "name": "anti-andran dev",
+        "command": "npm run dev"
+      }
     ]
   }
